generated/nifgen/nifgen_service.cpp

```diff
@@ -1217,6 +1233,39 @@ namespace nifgen_grpc {
     }
   }
 
+  //---------------------------------------------------------------------
+  //---------------------------------------------------------------------
+  ::grpc::Status NiFgenService::CreateWaveformComplexF64(::grpc::ServerContext* context, const CreateWaveformComplexF64Request* request, CreateWaveformComplexF64Response* response)
+  {
+    if (context->IsCancelled()) {
+      return ::grpc::Status::CANCELLED;
+    }
+    try {
+      auto vi_grpc_session = request->vi();
+      ViSession vi = session_repository_->access_session(vi_grpc_session.id(), vi_grpc_session.name());
+      ViConstString channel_name = request->channel_name().c_str();
+      ViInt32 number_of_samples = request->waveform_data_array().size();
+      auto waveform_data_array_request = request->waveform_data_array();
+      std::vector<NIComplexNumber_struct> waveform_data_array;
+      std::transform(
+        waveform_data_array_request.begin(),
+        waveform_data_array_request.end(),
+        std::back_inserter(waveform_data_array),
+        [&](nifgen_grpc::NIComplexNumber x) { return GetStructFromGrpcType(x); });
+
```

Comment:
Probably not a big deal, but can you check where this blank line is coming from? I didn't see any blank line in mako file where you are generating `transform` code.

Reply:
I added a backslash in the mako file after the `transform` code and this line was removed.